articles/users/search/v3/query-syntax.md

```diff
@@ -28,17 +28,30 @@ The query string is parsed into a series of terms and operators:
 
 ## Searchable fields
 
-You can search for users using the following fields:
-
-* All the [normalized user profile](/users/normalized/auth0/normalized-user-profile-schema) fields.
-* User metadata fields may be used with:
-    * Booleans
-    * Numeric (integer or double)
-    * Text
-    * Objects. In order to search a scalar value nested in another object, use the path to the field. For example, `app_metadata.subscription.plan:"gold"`
-    * Arrays. In order to search fields in objects nested arrays, use the path to the field and ignore the array level. For example, `user_metadata.addresses.city:"Paris"`
-
-Range and wildcard searches are not available on [user metadata](/users/concepts/overview-user-metadata) fields.
+You can search for users using the all the [normalized user profile](/users/normalized/auth0/normalized-user-profile-schema) fields:
+
+Field | Data Type
+------|------
+`phone_number` | text
```

Comment:
Instead of the data type representation, I think it will be better adding a more meaningful description, you can copy from here: https://auth0.com/docs/users/references/user-profile-structure

Reply:
Added descriptions